package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@netdata/dashboard",
-  "version": "1.5.4",
+  "version": "2.0.0-alpha.7",
```

Comment:
if this is `alpha`, perhaps we should have separate branch for this? `master` is used to release new versions for the Agent

Reply:
Yep, won't be merged until it reaches a stable version.